### ISSUE TITLE
[Question] How i can compile TIC-80 as a Libretro core for Nintendo Wii?

### ISSUE THREAD

**Issue by saulfabregwiivc** [nesbox/TIC-80#2552]
Hi!!!!

I'm interested in having a Wii homebrew core of TIC-80 for RetroArch.

And before you say me; yeah i know there's the official Libretro RetroArch releases but newer releases didn't care about the Wii, resulting in multiple issues and errors.
So i want to compile the latest improvements of the core, but using an older RetroArch for Wii which i know it works 100%.
I'd like to see how i can compile the core (the .a file) of TIC-80, seeing that i need to use CMake (which i have in my PC).

What is the CMake command for compile this for Wii?

Thanks.

**Comment by Miguel-hrvs** [comment]
Maybe compiling the source of the version that you like adding what you want could be what you're looking for: https://docs.libretro.com/development/retroarch/compilation/wii/

**Comment by saulfabregwiivc** [comment]
> Maybe compiling the source of the version that you like adding what you want could be what you're looking for: https://docs.libretro.com/development/retroarch/compilation/wii/ If I have time in a week or two I could play around with wii compilation.

yeah i know how to do that, but that is for cores which have standard Makefiles which are not CMake.
And i don't know how to compile TIC-80 in CMake for Libretro for Wii.

**Comment by Miguel-hrvs** [comment]
sdl2 was recently ported to the wii: https://gbatemp.net/threads/sdl-2-port-ongoing.645576/. So it would be better in terms of performance and stability to compile a standalone version
